novawallet/Common/Services/ExternalBalanceUpdater/CrowdloanService/CrowdloanOffChainSyncService.swift

```diff
@@ -73,6 +73,11 @@ final class CrowdloanOffChainSyncService: BaseSyncService {
     }
 
     override func performSyncUp() {
+        guard chain.syncMode.enabled() else {
```

Comment:
Let's omit that, as we agreed it is not intended that the service will run for disabled chain. Also, instead of stopSyncUp()